CMakeLists.txt

```diff
@@ -527,6 +527,22 @@ if(BUILD_TESTS)
       4000
   )
 
+  add_e2e_test(
+    NAME recovery_snapshot_test
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/recovery.py
+    CONSENSUS cft
+    ADDITIONAL_ARGS
+      --recovery
+      2
+      # Shorten Raft election timeout to speed up test when it kills a node on
+      # purpose to check that a recovery network is robust to a view change.
```

Comment:
We haven't a lot of timeouts due to quote verification since the dcap client started caching more aggressively, so I wonder if we shouldn't switch to a shorter election timeout across all our tests anyway.

Reply:
Yes. I'll lower this in a separate PR so we can test the stability of this in isolation.